src/components/ProductBacklogListComponent.js

```diff
@@ -158,14 +158,9 @@ const ProductBacklogListComponent = ({ displayPopup, loggedIn }) => {
           //Fetch the project - might be better to implement a getProjectByName
           const projects = await db.getProjects();
 
-          let currProject = projects.find(
-            (project) => project.ProjectName === state.projectName
-          );
-
-          if (currProject)
-            sessionStorage.setItem('project', JSON.stringify(currProject));
+          let currProject = projects.find((project) => project.ProjectName === state.projectName);
 
-          displayPopup('Successfully updated product backlog');
```

Comment:
Are we moving away from displaying popups?

Reply:
I think this one just needs to be reworked. As is, it was popping any time you loaded the page which was confusing and unnecessary feedback. I think it may be worthwhile to have in, but the logic for popping it on loading the component and actually updating the data needs to be separated

Reply:
Fair enough, that makes sense. There isnt much that you can do with it in its current state without adding more bools. The useEffect will always be hitting 3 times (once when it is created, once when it is initialized and once when it first gets assigned)